src/us/kbase/workspace/database/mongo/MongoWorkspaceDB.java

```diff
@@ -1581,14 +1622,6 @@ private void saveObjectVersions(final WorkspaceUser user,
 			v.put(Fields.VER_WS_ID, wsid.getID());
 			v.put(Fields.VER_ID, objectid);
 			v.put(Fields.VER_VER, ver++);
-			if (!v.containsKey(Fields.VER_TYPE_NAME)) {
```

Comment:
Originally my idea was to allow an online schema update post server upgrade, but then I realized that would mean any `list_objects` calls with a type filter would have missing data until the update was complete. As such, the server will be modified not to start until the update is complete, and this code isn't necessary.